src/test/regress/sql/multi_explain.sql

```diff
@@ -6,6 +6,9 @@
 ALTER SEQUENCE pg_catalog.pg_dist_shardid_seq RESTART 570000;
 ALTER SEQUENCE pg_catalog.pg_dist_jobid_seq RESTART 570000;
 
+-- print major version to make version-specific tests clear
+SHOW server_version \gset
+SELECT substring(:'server_version', '\d+\.\d+') AS major_version;
 
```

Comment:
> ```SELECT substring(version(), '\d+\.\d+');``` is easier to understand

Hm. Possibly, but I was worried about that function not being as well-structured (is it guaranteed the version is always the first number we'll come across?)…

I'll switch to one or the other that you suggest.